package/MDAnalysis/coordinates/PDB.py

```diff
@@ -1000,6 +998,23 @@ def CONECT(self, conect):
         self.pdbfile.write(self.fmt['CONECT'].format(conect))
 
 
+class PrimitivePDBReader(PDBReader):
```

Comment:
This class and the next one were removed in a previous commit. They should not be reintroduced. Please [rebase your branch](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) against the develop branch so it is up to date.